cmd/dapr.go

```diff
@@ -41,13 +41,26 @@ var RootCmd = &cobra.Command{
 Distributed Application Runtime`,
 }
 
-var logAsJSON bool
+type daprVersion struct {
+	CLI     string `json:"Cli version"`
+	Runtime string `json:"Runtime version"`
+}
+
+var (
+	daprVer   daprVersion
+	logAsJSON bool
+)
 
 // Execute adds all child commands to the root command.
 func Execute(version, apiVersion string) {
 	RootCmd.Version = version
 	api.RuntimeAPIVersion = apiVersion
 
+	daprVer = daprVersion{
+		CLI:     version,
+		Runtime: strings.ReplaceAll(standalone.GetRuntimeVersion(), "\n", ""),
```

Comment:
Any way we can make GetRuntimeVersion return values without "\n", replacing "\n" seems to be little hacky? Any thogughts?

Reply:
Currently `standalone.GetRuntimeVersion()` returns an extra newline. Need to see if it is a good idea to fix there.
But it could be done with a different PR imo.